src/gui/widgets/LcdWidget.cpp

```diff
@@ -140,15 +139,22 @@ void LcdWidget::paintEvent( QPaintEvent* )
 //	p.translate( width() / 2 - lcdWidth / 2, 0 ); 
 	p.save();
 
-	p.translate( margin, margin );
-
-	// Left Margin
-	p.drawPixmap( cellRect, *m_lcdPixmap, 
-			QRect( QPoint( charsPerPixmap*m_cellWidth, 
-				isEnabled()?0:m_cellHeight ), 
-			cellSize ) );
+	// Don't skip any space and don't draw margin on the left side in seamless mode
+	if (m_seamlessLeft)
+	{
+		p.translate(0, margin);
+	}
+	else
+	{
+		p.translate(margin, margin);
+		// Left Margin
+		p.drawPixmap( cellRect, *m_lcdPixmap,
+				QRect( QPoint( charsPerPixmap*m_cellWidth,
+					isEnabled()?0:m_cellHeight ),
+				cellSize ) );
```

Comment:
Just another way to split it up to be more readable with tabs. Should be fixed though. Also fix whitespace issues.
```suggestion
		p.drawPixmap(
			cellRect,
			*m_lcdPixmap,
			QRect(
				QPoint(
					charsPerPixmap * m_cellWidth,
					isEnabled() ? 0 : m_cellHeight
				),
				cellSize
			)
		);
```